velodyne_driver/launch/nodelet_manager.launch

```diff
@@ -14,6 +14,7 @@
   <arg name="repeat_delay" default="0.0" />
   <arg name="rpm" default="600.0" />
   <arg name="npackets" default="30" />
+  <arg name="gps_time" default="false" />
```

Comment:
why is gps_time false by default?

Reply:
Because you better be sure you have provided the velodyne NMEA data and your PC is synchronized to this time.